include/libbase64.h

```diff
@@ -53,6 +53,7 @@ extern "C" {
 #define BASE64_FORCE_SSE41	(1 << 5)
 #define BASE64_FORCE_SSE42	(1 << 6)
 #define BASE64_FORCE_AVX	(1 << 7)
+#define BASE64_CHECK_SUPPORT	(1 << 15)
```

Comment:
This flag is not documented in `README.md`. I also don't understand why the bitmasks make a numerical jump.

Reply:
I'm not sure if the flag makes sense. I would expect that even when the user forces a codec, we use runtime feature detection to check whether the codec is actually available. I can't see the use case for wanting to force an AVX2 codec on some low power machine and expecting anything sane to happen. If anything, the sane thing would be to return `false`.

Reply:
True, but that is what the comment said (force even if there is no CPU support) and I was not sure if I would be disturbing any test case by changing the default behavior. Then the bitmask is misused (not to add a codec but to change behavior), so I thought to leave room for future codecs.

But from my pov it would be best to not add the bitmask at all. So, if you agree...